core_integrations/hddtemp

```diff
@@ -0,0 +1 @@
+_homeassistant/
```

Comment:
HDD Temp is not an internal integration of Home Assistant.

Reply:
Ok, I will get a different icon